release/ray_release/schema.json

```diff
@@ -53,6 +53,9 @@
 				},
 				"alert": {
 					"type": "string"
+				},
+				"affected": {
+					"type": "string"
```

Comment:
could you split this file change into a separate pr?

Reply:
This is just a temporary change to kickoff a subset of release tests. Will revert this stuff later.

Reply:
ok.. then this PR does not need my approval :)